modules/consensus/accept_bench_test.go

```diff
@@ -0,0 +1,44 @@
+package consensus
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+)
+
+// benchmarkEmptyBlocks is a benchmark that mines many blocks, and
+// measures how long it takes to add them to the consensusset
+func benchmarkEmptyBlocks(bt *testing.B) error {
+	bt.StopTimer()
+
+	// The test dir will be reset each time the benchmark
+	// is done.
+	cst, err := createConsensusSetTester("BenchmarkEmptyBlocks")
+	if err != nil {
+		return errors.New("Error creating tester: " + err.Error())
+	}
+
+	for j := 0; j < bt.N; j++ {
+		b, _ := cst.miner.FindBlock()
+		bt.StartTimer()
+		err := cst.cs.AcceptBlock(b)
+		if err != nil {
+			errstr := fmt.Sprintf("Error accepting %d: %s", j, err.Error())
+			return errors.New(errstr)
+		}
+		cst.csUpdateWait()
+		bt.StopTimer()
```

Comment:
I thought about this test a bit more, and I think it's just in the wrong package. AcceptBlock, because of the subscription stuff, necessarily impacts all of the other modules, and therefore isn't very useful for benchmarking the consensus package alone.

`siad` might be a better place for this benchmark.